src/client/QXmppCall.h

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (C) 2008-2019 The QXmpp developers
+ *
+ * Author:
+ *  Jeremy Lainé
+ *
+ * Source:
+ *  https://github.com/qxmpp-project/qxmpp
+ *
+ * This file is a part of QXmpp library.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ */
+
+#ifndef QXMPPCALL_H
+#define QXMPPCALL_H
+
+#include <QObject>
+#include <QMetaType>
+
+#include "QXmppCallStream.h"
+#include "QXmppClientExtension.h"
+#include "QXmppLogger.h"
+#include "QXmppCallStream.h"
+
+class QHostAddress;
+class QXmppCallPrivate;
+class QXmppCallManager;
+class QXmppCallManagerPrivate;
+
+/// \brief The QXmppCall class represents a Voice-Over-IP call to a remote party.
+///
+/// \note THIS API IS NOT FINALIZED YET
```

Comment:
Please add `\since QXmpp 1.2` to all new classes or new methods.

Reply:
I added it to the new QXmppCallStream class and the new methods in QXmppCall. Is that enough or should I also add it to all new methods in QXmppCallStream?

Reply:
I think it's enough to only add it to the class documentation when the class is new.